wordpress-mobile/gutenberg-mobile
# Moves hermes and react-native mirror to S3

See https://github.com/WordPress/gutenberg/pull/31441.

This PR updates `react-native-bridge`, `react-native-aztec` and `react-native-editor` projects to use react native mirror and hermes binaries from S3 repo instead of Bintray. This change is made due to [Bintray shutting down](https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/).

To test:
- See gutenberg PR link above
- run the demo app locally
- Verify CI was able to build and publish binary

PR submission checklist:
